### PR TITLE
Change version for Gtk.TextViewLayer 

### PR DESCRIPTION
**Pull request by foo-jin**
The fix as suggested in #653.

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -1962,3 +1961,8 @@ status = "generate"
     #value glib.Variant
     name = "action-target"
     ignore = true
+
+[[object]]
+name = "Gtk.TextViewLayer"
+status = "generate"
+version = "3.14"
```
Comment (EPashkin) [inline Gir.toml]: Please, move this between Gtk.TextView and Gtk.Toolbar.

Comment (GuillaumeGomez) [reply Gir.toml]: We should add a test to enforce that it's alpha sorted in the toml file. I'm going to add when I have a bit of time. :)

**Comment by EPashkin** [comment]
@GuillaumeGomez Yes no version in docs, but enum not present in 3.10 libs and used only in internal `GtkTextViewClass.draw_layer`, so it safe to say that it added with it.

**Comment by GuillaumeGomez** [comment]
That's weird... But whatever, let's merge it then! Thanks!

**Comment by EPashkin** [comment]
Hm, missed that it not against master.
I will repeat  on master

**Comment by GuillaumeGomez** [comment]
Wait what?! Oh damn! I remove the commit from history.

**Comment by GuillaumeGomez** [comment]
Ok done. Please reopen it against the master branch (or maybe not needed if @EPashkin does it?).

**Comment by EPashkin** [comment]
@foo-jin I replaced PR. Thanks for reporting problem.

**Pull request merged**